lib/portlayer/storage/vsphere/image.go

```diff
@@ -41,7 +42,22 @@ import (
 var StorageParentDir = "VIC"
 
 // Set to false for unit tests
-var DetachAll = true
+var (
+	DetachAll = true
+
+	FileForMinOS = map[string]string{
```

Comment:
`map[string]os.FileMode` - means we don't have to do the runtime conversion all the time and cleans up the loops.